functions/gooey_builder_deploy_tool.py

```
@@ -139,19 +88,31 @@ def call(
         twilio_country_code: str | None = None,
         whatsapp_country_code: str | None = None,
         telegram_bot_token: str | None = None,
+        run_url: str | None = None,
     ) -> dict:
+        page_cls, sr, pr = self.get_current_runs(run_url)
```

Ruff (0.16.0)

[warning] Unpacked variable `page_cls` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)